java/code/src/com/suse/manager/webui/controllers/admin/handlers/test/PaygApiControllerTest.java

```diff
@@ -88,7 +86,7 @@ private void clearDb() {
         HibernateFactory.commitTransaction();
     }
 
-    private PaygSshData createPaygSshData() {
+    private PaygSshData createPaygSshData() throws Exception {
```

Comment:
Instead of throwing a generic exception should it be a more concrete one?

Reply:
This was in a "Test", so it was easier than adding 3 different exceptions. But I can also make the list.